lms/static/images/icon-cert-white.svg

```diff
@@ -0,0 +1,10 @@
+<svg id="Layer_2" data-name="Layer 2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12.85 9.22">
```

Comment:
I would remove the id and data-name elements, I don't think they offer us any value.